lib/ansible/modules/system/puppet.py

```diff
@@ -192,6 +197,7 @@ def main():
             summarize=dict(type='bool', default=False),
             debug=dict(type='bool', default=False),
             verbose=dict(type='bool', default=False),
+            no_use_srv_records=dict(type='bool', default=False),
```

Comment:
I would change it in a way to toggle either`--use_srv_records` or --no-use_srv_records` whether this value is true or false

~~~
 no_use_srv_records=dict(type='bool'),